MAINTAINERS.md

```diff
@@ -0,0 +1,78 @@
+# Maintainers
+
+## Active Maintainers
+
+<!-- Please keep this sorted alphabetically by github -->
+
+| Name          | Github      | LFID |
+| ------------- | ----------- | ---- |
+| Akiff Manji   | amanji      |      |
+| Clecio Varjao | cvarjao     |      |
+| James Ebert   | JamesKEbert |      |
+| Jason Leach   | jleach      |      |
+| Ryan Koch     | ryanindicio |      |
```

Comment:
From what I can see in the Hyperledger Teams, this is not the right list. It should be:

| Name          | Github      | LFID |
| ------------- | ----------- | ---- |
| @jleach	Jason C. Leach | jleach | |
| @TelegramSam	Sam Curren | TelegramSam | |
| @cvarjao	Clécio Varjão | cvarjao | |
| @amanji	Akiff Manji | amanji | |
| @karimStekelenburg	Karim Stekelenburg | karimStekelenburg | |
| @TimoGlastra	Timo Glastra | TimoGlastra | |
| @JamesKEbert	James Ebert | JamesKEbert | |

See: https://github.com/orgs/hyperledger/teams?query=aries-mobile-agent

Reply:
Not sure those are the right people -- just that those are the ones on the team.

@ryjones -- a general question. On the "committers" team, James is "Maintainer". However, I believe the definition that Hyperledger uses for Maintainer is really a "Committer" -- someone that can approve PRs and merge them.  Is that correct?

Reply:
@swcurran On the last Bifold WG call @JamesKEbert proposed the base policy from which I went with based on the [MAINTAINTERS.md](https://github.com/hyperledger/anoncreds-spec/blob/main/MAINTAINERS.md) from the AnonCreds Spec repo. It outlined some guidelines for becoming and removing maintainers. When I created the list I roughly applied those guidelines for my decisions.

Based on that, Karim and Timo haven't contributed since 2021 so I'm hesitant to not follow the guidelines.

Reply:
@swcurran the "maintainer" in that role means they can add people to that team in the GH UI